content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,221 @@
+:show-content:
+
+======
+Orders
+======
+
+When the *Lunch* application is opened, the :guilabel:`Order Your Lunch` dashboard loads. This view
+is also accessed by navigating to :menuselection:`Lunch app --> My Lunch: New Order`.
+
+The :guilabel:`Order Your Lunch` dashboard provides a summary of lunch offerings, the user's account
+information, the current day's orders, and their statuses.
+
+Order Your Lunch
+================
+
+On the main :guilabel:`Order Your Lunch` dashboard, all the necessary information needed to place an
+order is visible. The default filter for the products is :guilabel:`Available Today`. This filter
+shows only products that can be purchased that day, based on the :ref:`vendor's availability
+<lunch/availability>`.
+
+The left-side of the dashboard displays the various :guilabel:`Categories` of products available,
+along with the :guilabel:`Vendors` supplying the products. On the far-right side of each line is a
+number, which indicates how many products are associated with that respective category or vendor.
+
+To filter the products by categories or vendors, click the checkbox next to the specific category or
+vendor to view only those items.
+
+.. note::
+   If multiple selections are made, **only** products that fall under **all** the selected options
+   are shown.
+
+The top portion of the dashboard, the :guilabel:`Your Order` summary, displays the user's account
```

Comment:
UPDATE: So, I've come to see that "Your Order" does, indeed, appear at the top -- but **only** when you add a product to an order/have one that you're building...It does not, however, appear by default when you open the application. Therefore, I *still* think we should opt for the vague explanation, sans GUI, because that reveals itself later on in the doc. This way, readers will still understand what's going on, without any potential confusion.